### DISCUSSION TITLE
MongoDB not installable on Ubuntu 22.04 so MeshCentral Security Can't Be Improved

### DISCUSSION THREAD

**Discussion by compumatter** [Ylianst/MeshCentral#4954] [General]
Hello,

First I would like to thank you for your significant effort and what appears to be a wonderful product.  I hope you are able to maintain it and keep it going in some way, shape or form.

Here is my question...

This page indicates installing mongodb will improve security:
* https://meshcentral-community.com/doku.php?id=howto:installation:manual:ubuntu:moresecure&s[]=security

However, mongodb requires openssl 1.1 which has known security vulnerabilities when used on 22.04 as discussed here https://lists.ubuntu.com/archives/ubuntu-security-announce/2022-May/006538.html

So the improved security method comes with a security problem if openssl 1.1 is force installed.

So I am wondering if this is being addressed in any way?

I know there has been talk of alternative database support a few years ago but it doesn't appear to be moving forward.   MongoDB is not keeping up with the times at least not on their community version.  This is keeping MeshCentral at that same 20.04 limitation for its own security.

What do you think / suggest ?

Jay / CompuMatter

**Comment by dinger1986** [comment]
there are other databases available, postgresql, mysql, mariadb are some I have used in the past

**Comment by compumatter** [comment]
Yes but are they supported by MeshCentral ?   I don't think so.Sincerely,JayJay LeporeCompuMatterOffice: 360 299 2428Cell: 360 202 ***@***.*** you are satisfied with your service, please consider providing us a social media review at https://compumatter.com/reviews...If you are not, please let me know so I can make it betterOn Feb 3, 2023, at 1:42 AM, dinger1986 ***@***.***> wrote:﻿
there are other databases available, postgresql, mysql, mariadb are some I have used in the past

**Reply by dinger1986** [reply]
yes! https://ylianst.github.io/MeshCentral/meshcentral/config/

**Reply by compumatter** [reply]
Wow.   Meshcentrals best kept secret !

**Reply by dinger1986** [reply]
It's been available for afew years. Or at least some of it has!
